cmd/cosign/cli/verify/verify_attestation.go

```diff
@@ -275,9 +275,10 @@ func (c *VerifyAttestationCommand) Exec(ctx context.Context, images []string) (e
 
```

Comment:
> unsupported: trusted root path currently only supported with --new-bundle-format

I don't think this is true anymore, the trusted material loaded from the trusted root path at the beginning of this file is now useful for verifying any bundle https://github.com/sigstore/cosign/blob/c36801ef34aa24eb01d064ab12da66a83b9c3ff4/pkg/cosign/verify.go#L1192-L1206

so I think this whole switch case could be deleted.